src/99-appendices/10-file-collections.md

```diff
@@ -0,0 +1,53 @@
+| TB1EPI | flip, echo                    | B1<sup>+</sup> mapping with 3D EPI | This B1<sup>+</sup> mapping method ([Jiru and Klose 2006](https://dx.doi.org/10.1002/mrm.21083)) is based on two EPI readouts to acquire spin echo (SE) and stimulated echo (STE) images at multiple flip angles in one sequence, used in the calculation of deviations from the nominal flip angle.                                                                              |
+| TB1AFI | flip, inv                     | Actual Flip Angle Imaging (AFI)    | This method ([Yarnykh 2007](https://dx.doi.org/10.1002/mrm.21120)) calculates a B1<sup>+</sup> map from two images acquired at interleaved (two) TRs with identical RF pulses using a steady-state sequence                                                                                                                                                                       |
+| TB1TFL | Please see the qMRI appendix* | Siemens `tfl_b1_map`               | B1<sup>+</sup> data acquired using `tfl_b1_map` product sequence by Siemens based on the method by [Chung et al. (2010)](https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.22423). The sequence generates one ~anatomical image and one scaled flip angle map.                                                                                                                 |
+| TB1RMF | Please see the qMRI appendix* | Siemens `rf_map`                   | B1<sup>+</sup> data acquired using `rf_map` product sequence by Siemens                                                                                                                                                                                                                                                                                                           |
+| RB1COR | Please see the qMRI appendix* | B1<sup>-</sup> field correction    | Low resolution images acquired by the body coil (in the gantry of the scanner) and the head coil using identical acquisition parameters to generate a combined sensitivity map as described in [Papp et al. (2016)](https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.26058)                                                                                                   |
+
+\* These file collections call for the use of special notations that cannot
+be resolved by entities that can generalize to other applications. Instead of
+introducing an entity that is exclusive to a single application, method developers
+who commonly use these file collections for the `MPM` application reached the consensus
+on the use of `acq` entity to distinguish individual files. Please visit the qMRI
+appendix for further details.
```

Comment:
I think an option to solve https://github.com/bids-standard/bids-specification/pull/683#discussion_r531695745 would be to take this last paragraph starting with `\*` and put it at the top of the qMRI appendix, with a note right below, that more information will be added in the future.

Then in the table above all the `*` footnote references (which is broken markdown BTW, because it expects italics after an unescaped  `*` chat) can be removed, and `Please see the qMRI appendix` can be turned into a true link instead.